packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -396,6 +416,24 @@ class _DraggableState<T> extends State<Draggable<T>> {
   }
 }
 
+/// Represents the draggable's current details, such as whether the
+/// [DragTarget] accepted it, its [Velocity], and its [Offset].
+class DraggableDetails {
+  DraggableDetails._({
+    this.wasAccepted,
+    this.velocity,
+    this.offset});
+
+  /// Determines whether the [DragTarget] accepted this draggable
+  final bool wasAccepted;
+
+  /// This draggable's current [Velocity]
+  final Velocity velocity;
+
+  /// This draggable's current [Offset]
+  final Offset offset;
```

Comment:
For these docs we should have more detail (these don't technically pass our style guide, since they don't give more information than you could guess from the names). Please see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-comments-dartdocs for discussion including a list of writing prompts that might suggest more information to include here.

Reply:
@Hixie added some more details to the documentation of my changes. Please let me know if this suffices, since this should be in line with style guidelines.